### PR TITLE
BDLaethg39

### PR DESCRIPTION
**Pull request by thea-m**
Last JJ scroll from my side :)

### CODE REVIEW

File: OxfordBodleian/Juel-Jensen/BDLaethg39.xml [hunk 1]
```diff
@@ -53,25 +210,184 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
...
+                                    <height>85</height>
+                                    <width>87</width>
+                                </dimensions>
+                            </decoNote>
                         </decoDesc>-->
                         
+                        <additions>
+                            <list>
+                                <item xml:id="e1">
+                                    <desc>The verso of the scroll seems to mirror its recto: It contains
```
Comment (MassimoVilla) [inline OxfordBodleian/Juel-Jensen/BDLaethg39.xml:357]: perhaps" contained" since neither the miniatures nor the text are now clearly visible?

**Review by MassimoVilla — CHANGES_REQUESTED** [review]
Thank you Thea, just one very minor observation

File: OxfordBodleian/Juel-Jensen/BDLaethg39.xml (hunk 1 above)
Comment (gnizla) [reply OxfordBodleian/Juel-Jensen/BDLaethg39.xml:357]: Sorry, but I am confused by this: is this not simply the ink that has penetrated into the parchment due to e.g. dampness?

Comment (thea-m) [reply OxfordBodleian/Juel-Jensen/BDLaethg39.xml]: You are completely right, thank you!
I included the information in condition (which I had previously forgotten...).

**Review by MassimoVilla — APPROVED** [review]

**Review by ElianaDalSasso — APPROVED** [review]

**Review by gnizla — APPROVED** [review]

**Pull request merged**